### PR TITLE
Add loki datasource and dashboard to grafana

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

- Add loki datasource to grafana
- add dashboard to grafana

### CODE REVIEW

**Comment by wolfstudy** [comment]
@sijie @codelipenghui PTAL

**Comment by wolfstudy** [comment]
image

File: Dockerfile.kubernetes [hunk 1]
```diff
@@ -56,6 +56,8 @@ COPY entrypoint.sh /pulsar_grafana_entrypoint.sh
 EXPOSE 3000
 
 ENV PULSAR_PROMETHEUS_URL http://localhost:9090
+ENV GF_LOKI_URL http://sn-platform-pulsar-loki:3100
```
Comment (sijie) [inline Dockerfile.kubernetes:59]:
```suggestion
ENV GF_LOKI_URL http://localhost:3100
```

File: Dockerfile [hunk 2]
```diff
@@ -57,6 +57,8 @@ EXPOSE 3000
 
 ENV PULSAR_PROMETHEUS_URL http://localhost:9090
 ENV PULSAR_CLUSTER pulsar-cluster
+ENV GF_LOKI_URL http://sn-platform-pulsar-loki:3100
```
Comment (sijie) [inline Dockerfile:60]:
```suggestion
ENV GF_LOKI_URL http://localhost:3100
```

**Pull request merged**